net-mgmt/lldpd/src/opnsense/service/templates/OPNsense/Lldpd/lldpd.conf

```diff
@@ -0,0 +1,2 @@
+{% if helpers.exists('OPNsense.lldpd.general.enabled') and OPNsense.lldpd.general.enabled == '1' %}
```

Comment:
will this ever receive content?

Reply:
Yes, you can set Hostname etc. to manipulate the default values, but for version 1.0 it's not really required, we'll see if there's a need for it.